cmake/catkin_python_setup.cmake

```diff
@@ -163,6 +140,34 @@ function(catkin_python_setup)
       TARGET_NAME ${PROJECT_NAME}_${name}_exec_install
       DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
   endforeach()
+
+  if(${PROJECT_NAME}_SETUP_PY_SETUP_MODULE STREQUAL setuptools)
+    message(STATUS "Appending egg base")
```

Comment:
While this was useful for debugging I would suggest to remove it from the patch.

Reply:
Resolved in 0c9d05d4